src/seedu/addressbook/AddressBook.java

```diff
@@ -133,22 +134,25 @@
     private static final String COMMAND_EXIT_DESC = "Exits the program.";
     private static final String COMMAND_EXIT_EXAMPLE = COMMAND_EXIT_WORD;
 
+    private static final int ARGUMENTS_ZERO = 0;
+    private static final int ARGUMENTS_ONE = 1;
+    private static final int ARGUMENTS_TWO = 2;
+    private static final int ARGUMENTS_THREE = 3;
```

Comment:
Good to see you removing the magic number, however you replace it with "magic words".. :) It would be better if you can come up with better naming for these constants.